diff --git a/node_modules/react-native-bignumber/cpp/MGBigNumberHostObject.cpp b/node_modules/react-native-bignumber/cpp/MGBigNumberHostObject.cpp
index 9e8dd7e..1c57b76 100644
--- a/node_modules/react-native-bignumber/cpp/MGBigNumberHostObject.cpp
+++ b/node_modules/react-native-bignumber/cpp/MGBigNumberHostObject.cpp
@@ -23,7 +23,7 @@ namespace margelo
     {
         this->fields.push_back(HOST_LAMBDA("createFromString", {
             std::string strRep = arguments[0].getString(runtime).utf8(runtime);
-            strRep.erase(std::remove_if(strRep.begin(), strRep.end(), std::isspace), strRep.end());
+            strRep.erase(remove_if(strRep.begin(), strRep.end(), isspace), strRep.end());
             int base = 10;
             if (!arguments[1].isUndefined())
             {
diff --git a/node_modules/react-native-bignumber/react-native-bignumber.podspec b/node_modules/react-native-bignumber/react-native-bignumber.podspec
index d9fbaea..d8e6d38 100644
--- a/node_modules/react-native-bignumber/react-native-bignumber.podspec
+++ b/node_modules/react-native-bignumber/react-native-bignumber.podspec
@@ -32,7 +32,7 @@ Pod::Spec.new do |s|
     "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_TARGET_SRCROOT)\"  \"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/Headers/Private/React-Core\" "
   }
   s.xcconfig               = {
-    "CLANG_CXX_LANGUAGE_STANDARD" => "c++14",
+    "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
     "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/glog\"  \"${PODS_ROOT}/Headers/Public/React-hermes\" \"${PODS_ROOT}/Headers/Public/hermes-engine\""
   }
 
